src/core/server/logging/README.md

```diff
@@ -324,12 +327,33 @@ Here is what we get with the config above:
 | server           | console, file            | fatal |
 | optimize         | console                  | error |
 | telemetry        | json-file-appender       | all   |
+| metrics.ops      | console                  | debug |
+
 
+### Dedicated loggers
 
 The `root` logger has a dedicated configuration node since this context is special and should always exist. By 
 default `root` is configured with `info` level and `default` appender that is also always available. This is the 
 configuration that all custom loggers will use unless they're re-configured explicitly.
 
+The `metrics.ops` logger is configured with `debug` level and will automatically output sample system and process information at a regular interval.
+The metrics that are logged are a subset of the data collected and are formatted in the log message as follows:
+
+| Ops formatted log property | Location in metrics service | Log units
+| :------------------------- | :-------------------------- | :-------------------------- |
+| memory | process.memory.heap.used_in_bytes | [depends on the value](http://numeraljs.com/#format), typically MB or GB |
+| uptime  | process.uptime_in_millis | HH:mm:ss |
+| load  | os.load | [ "load for the last 1 min" "load for the last 5 min" "load for the last 15 min"] |
+| delay | process.event_loop_delay | ms |
+
+The log interval is the same as the interval at which system and process information is refreshed and is configurable under `ops.interval`:
+
+```yaml
+ops.interval: 5000
+```
+
+The minimum interval is 100ms and defaults to 5000ms.
```

Comment:
I think this whole section would make more sense after the last few paragraphs below ("For example to see _all_ log messages that fall back on the `root` logger configuration...").

Currently if you are reading the document top-to-bottom it is a bit awkward since the first section on the `root` logger is split up.

Reply:
I see what you mean and agree. I'll change it.